deno/compress_image/src/mod.ts

```diff
@@ -0,0 +1,77 @@
+import { base64Encode } from "./deps.ts";
+
+export default async function (req: any, res: any) {
+  const { provider, image } = JSON.parse(req.payload);
+  switch (provider) {
+    case "tinypng":
+      if(!req.variables.TINIFY_API_KEY) {
+        return res.json({
+          success: false,
+          message: "The environment variables (TINIFY_API_KEY) are required."
+        })
+      }
+
+      try {
+        const receiveImage = await fetch(image);
```

Comment:
The image input is a base64 encoded image. Passing that to this results in an error:

> TypeError: Invalid URL

Please fix this to accept a base64 encoded image.

Reply:
@MoskalykA right, this function won't actually work in Appwrite. You would run it manually with open runtimes just as instructed in the readme.

Reply:
Agreed